etc/picongpu/taurus-tud/picongpu.profile.example

```diff
@@ -14,7 +13,7 @@ module load gnuplot/4.6.1
 ### GCC
 module load gcc/5.3.0
 echo "WARNING: Boost version is too old! (Need: 1.62.0+)" >&2
-# module load boost/1.60.0-gnu5.3
+# module load boost/1.64.0-gnu5.3
```

Comment:
This was just a reminder for myself to check this before merge again. 
(So in case that the module is available, the `#` could be removed.)